src/Microsoft.AspNetCore.Server.IIS/Core/IISHttpContext.cs

```diff
@@ -191,6 +197,38 @@ public string ReasonPhrase
             }
         }
 
+        public unsafe X509Certificate2 ClientCertificate
+        {
+            get
+            {
+                if (_options.ForwardClientCertificate &&
```

Comment:
I think we can remove ForwardClientCertificate since ClientCertificate itself is lazy.